src/layout/page/pageFiltersRail.jsx

```diff
@@ -13,6 +13,10 @@ const propTypes = {
          * If `true`, filters are applied
          */
         isDirty: PropTypes.bool.isRequired,
```

Comment:
Should the comment for `isDirty` be changed?
Maybe something like:
> If `true`, then filter options have been modified but not yet applied to the list results.

Reply:
Fixed. [Commit Link](https://github.com/saddlebackdev/react-cm-ui/pull/481/commits/cad8178f27e4cf0a81c6d50d27fdc4f7365e3199)

Reply:
Thanks.  Resolving.